ControlRoomApplication/ControlRoomApplication/GUI/DiagnosticsForm.cs

```diff
@@ -279,7 +279,16 @@ private void timer1_Tick(object sender, System.EventArgs e)
             _azEncoderDegrees = currAbsOrientation.Azimuth;
             _elEncoderDegrees = currAbsOrientation.Elevation;
             lblAzAbsPos.Text = Math.Round(_azEncoderDegrees, 2).ToString();
-            lblElAbsPos.Text = Math.Round(_elEncoderDegrees, 2).ToString();
+
+            // Check if elevation encoder is timed out, output ERR if so, otherwise output current position
+            if (rtController.RadioTelescope.SensorNetworkServer.SensorStatuses.ElevationAbsoluteEncoderStatus.Equals(Entities.DiagnosticData.SensorNetworkSensorStatus.Error))
```

Comment:
You can use `using ...` for the sensor statuses at the top of the page so you don't have to explicitly specify the pathing for the SensorNetworkSensorStatus. It makes it a little easier to read.

- [ ] Change to `using` at top of page